app/components/project-list/index.ts

```diff
@@ -33,20 +41,97 @@ export default class ProjectListComponent extends Component {
   @tracked query = DEFAULT_PROJECT_QUERY_PARAMS.query;
   @tracked sortKey = DEFAULT_PROJECT_QUERY_PARAMS.sortKey;
   @tracked platform = DEFAULT_PROJECT_QUERY_PARAMS.platform;
   @tracked team = DEFAULT_PROJECT_QUERY_PARAMS.team;
+  @tracked allColumnsMap: Map<string, FilterColumn>;
+  @tracked showColumnManager = false;
+  @tracked disableColumnManager = false;
 
   constructor(owner: unknown, args: object) {
     super(owner, args);
 
+    this.allColumnsMap =
+      this.loadColumnPreferences() ||
+      initializeColumns(DEFAULT_PROJECT_COLUMNS);
+
     this.projectService.fetchProjects.perform();
   }
 
+  loadColumnPreferences() {
+    try {
+      const saved = localStorage.getItem(COLUMN_PREFERENCES_KEY);
+      if (!saved) {
+        return null;
+      }
+
+      const preferences = JSON.parse(saved);
+      if (typeof preferences !== 'object' || preferences === null) {
+        return null;
+      }
+
+      // Get default columns as the base
+      const defaultColumnsMap = initializeColumns(DEFAULT_PROJECT_COLUMNS);
+      const map = new Map<string, FilterColumn>();
+
+      // Iterate through default columns and merge with saved preferences
+      defaultColumnsMap.forEach((defaultColumn, field) => {
+        const savedPref = preferences[field];
+
+        if (savedPref && typeof savedPref === 'object') {
+          // Merge: keep name, field, default, and base isHideable/isFixed from defaults
+          map.set(field, {
+            field: defaultColumn.field,
+            name: defaultColumn.name,
+            default: defaultColumn.default,
+            isHideable: defaultColumn.isHideable,
+            isFixed: defaultColumn.isFixed,
+            selected: savedPref.selected ?? defaultColumn.selected,
+            order: savedPref.order ?? defaultColumn.order,
+            width: savedPref.width ?? defaultColumn.width,
+            minWidth: savedPref.minWidth ?? defaultColumn.minWidth,
+          });
+        } else {
+          // No saved preference, use default
+          map.set(field, defaultColumn);
+        }
+      });
+
+      if (map.size !== defaultColumnsMap.size) {
+        console.warn('Saved preferences incomplete, resetting to defaults');
+        localStorage.removeItem(COLUMN_PREFERENCES_KEY);
+        return null;
+      }
+
+      return map;
+    } catch (error) {
+      console.error('Failed to load column preferences:', error);
+      localStorage.removeItem(COLUMN_PREFERENCES_KEY);
+      return null;
+    }
+  }
+
+  saveColumnPreferences() {
+    const preferences = Array.from(this.allColumnsMap.entries()).reduce(
+      (acc, [field, column]) => ({
+        ...acc,
+        [field]: {
+          selected: column.selected,
+          order: column.order,
+          width: column.width,
+          minWidth: column.minWidth,
+        },
+      }),
+      {}
+    );
+
+    localStorage.setItem(COLUMN_PREFERENCES_KEY, JSON.stringify(preferences));
+  }
   get isLoading() {
```

Comment:
Please move getters to just below the constructor followed by actions and undecorated methods.